tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__HandlerFactory.java

```diff
@@ -19,23 +22,22 @@
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 
 /**
- * The {@link ${bindingIdCamelCase}HandlerFactory} is responsible for creating things and thing 
+ * The {@link ${bindingIdCamelCase}HandlerFactory} is responsible for creating things and thing
  * handlers.
- * 
+ *
  * @author ${author} - Initial contribution
  */
 public class ${bindingIdCamelCase}HandlerFactory extends BaseThingHandlerFactory {
-    
+
     private final static Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_SAMPLE);
```

Comment:
pitty, that I missed it, it has been merged so it would be great if you could open a new pr for this one :-)